### ISSUE TITLE
Support for new features 

### ISSUE THREAD

**Issue by stroodle96** [elcajon-tech/addon-tailscale#16]
### The problem

Are there plans to support https certificates from tailscale? 

Also the funnel feature is in alpha, any plans to support that so we can use tailscale to make HA publicly accessible?

### What version of Tailscale has the issue?

N/A

### What type of installation are you running?

Home Assistant OS

### Steps to reproduce the issue

Inquiry.

**Comment by elcajon** [comment]
The current implementation supports [Caddy's proxy](https://tailscale.com/blog/caddy/) feature to access your HA via https out of the box without the need for any additional configuration. (You just need to enable HTTPS support for your Tailnet.)

Check the log for the following lines:

```
[22:15:36] INFO: Your Homeassistant instance is available via Tailscale.
[22:15:36] INFO: Your Domain is homeassistant.xxx.ts.net
[22:15:36] INFO: Starting Caddy...
```

I already played around with the unstable release of Tailscale which supports Funnel functionalities (see #11). I guess with the today released version 1.34.0 the function will be available in stable package (as alpha). I will take a closer look in the coming days. I assume I'll add another configuration option that adds the ability to make the already existing domain externally accessible. There is no plan (from my side, feel free to contribute) to add more complex proxy functionalities.

**Comment by elcajon** [comment]
The latest release contains support for Tailscale Funnels, if you are still interested. :)
